packages/core/documentation/src/components/DocumentationPageEmptyState.vue

```diff
@@ -2,29 +2,23 @@
   <KEmptyState
     class="documentation-page-empty-state"
     data-testid="documentation-page-empty-state"
-    icon="stateGruceo"
-    icon-size="96"
   >
+    <template #icon>
+      <KongIcon />
```

Comment:
@dustinryerson would [FileEmptyIcon](https://kong.github.io/icons/?q=FileEmptyIcon) icon work better here? Context: design identified `KongIcon` as best substitute for old `stateGruceo` icon but since this page is related to documentation empty state I thought maybe file icon would work better here.